### PR TITLE
[Doc Issue][Two sections on getting context with shared channels-- duplicate, or missing context?]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/access-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ddc6535127d9805651b605fdcef60b3c4d08022/msteams-platform%2Ftabs%2Fhow-to%2Faccess-teams-context.md) | 91 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32573e11-e5d4-4537-b890-add40be918a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e8211ff3d5e95776820cbb8d157c1f7f7610615/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f5a493a-fad2-41f4-986a-bc2fdfd8c04b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e8211ff3d5e95776820cbb8d157c1f7f7610615/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9c4580b7-b35c-4126-b545-e51fc29ca491) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e8211ff3d5e95776820cbb8d157c1f7f7610615/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 95 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bfc79f11-de83-4cf7-9ea7-438baaf4002f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/access-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e8211ff3d5e95776820cbb8d157c1f7f7610615/msteams-platform%2Ftabs%2Fhow-to%2Faccess-teams-context.md) | 91 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f6fd779-89f4-4af5-a70e-cd4f3a6b4fac) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/tabs/how-to/access-teams-context.md [hunk 1]
```diff
@@ -236,30 +236,11 @@ The following fields are changed when your content page is in a private channel:
...
-
-* `hostTeamGroupId`: Set to the `team.groupId` associated with the hosting team, or the team that created the shared channel. The property can make Microsoft Graph API calls retrieve membership of the shared channel.
-* `hostTeamTenantId`: Set to the `channel.ownerTenantId` associated with the hosting team, or the team that created the shared channel. The property can be cross-referenced with the current user's tenant ID found in the `user.tenant.id` field of the *context* object to determine if the user is internal or external to the hosting team's tenant.
-
-If your page makes use of any of these values, the value of `channel.membershipType` field must be `Shared` to determine if your page is loaded in a shared channel and can respond appropriately.
-
 > [!NOTE]
-> `teamSiteUrl` also works well for standard channels.
+>
+> * Currently, Microsoft Teams Connect shared channels are in developer preview only.
```
Comment (joeyglocke) [inline msteams-platform/tabs/how-to/access-teams-context.md:241]: This is actually no longer the case - it's GA

**Review by joeyglocke — CHANGES_REQUESTED** [review]
One comment inline

File: msteams-platform/tabs/how-to/access-teams-context.md (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/tabs/how-to/access-teams-context.md:241]: I have removed the note.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/98f6c4bb6829dd477dafb2cbd0ec1f3ade429268/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2cb94c3b-1d8d-46d6-9859-a5c84070bb1d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/98f6c4bb6829dd477dafb2cbd0ec1f3ade429268/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd61dbc1-68c6-481f-ba93-b1da052b947d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/98f6c4bb6829dd477dafb2cbd0ec1f3ade429268/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 95 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b7143377-8e8e-46a0-a3f4-48d7e31e289e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/access-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/98f6c4bb6829dd477dafb2cbd0ec1f3ade429268/msteams-platform%2Ftabs%2Fhow-to%2Faccess-teams-context.md) | 91 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9e7dfc3-86db-46a1-a4f2-e5d0da80d217) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bbe6133730ae26972c58002ead2f2305c55d65d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6aa4d87-c16f-4695-b1cf-9036d21d52c7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bbe6133730ae26972c58002ead2f2305c55d65d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eba0f07a-787d-4a97-a19e-2d75e33cb3d5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bbe6133730ae26972c58002ead2f2305c55d65d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 95 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4dd14be1-bbe4-4dce-9900-8c72e47375b7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/access-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bbe6133730ae26972c58002ead2f2305c55d65d/msteams-platform%2Ftabs%2Fhow-to%2Faccess-teams-context.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41672552-5b2b-432a-8155-6f64b8791a8f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by joeyglocke — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**